contracts/src/protocol/OmniAVS.sol

```diff
@@ -328,16 +333,41 @@ contract OmniAVS is IOmniAVS, IOmniAVSAdmin, IServiceManager, OwnableUpgradeable
 
     /// @inheritdoc IOmniAVSAdmin
     function addToAllowlist(address operator) external onlyOwner {
+        _addToAllowlist(operator);
+    }
+
+    /// @inheritdoc IOmniAVSAdmin
+    function removeFromAllowlist(address operator) external onlyOwner {
+        if (_isOperator(operator)) _deregsiterOperatorFromAVS(operator);
+        _removeFromAllowlist(operator);
+    }
+
+    /// @inheritdoc IOmniAVSAdmin
+    function ejectOperator(address operator) external onlyOwner {
+        require(_isOperator(operator), "OmniAVS: not an operator");
+
+        _removeFromAllowlist(operator);
+        _deregsiterOperatorFromAVS(operator);
+
+        emit OperatorEjected(operator);
+    }
```

Comment:
It seems like `removeFromAllowList` and `ejectOperator` are basically the same - except

- `ejectOperator` can only be called when an operator is already an operator
- `removeFromAllowList` can be called before an operator is an operator

Should we just join these into one from function?